src/misc/lv_event.c

```diff
@@ -54,6 +54,19 @@ static lv_event_dsc_t ** event_array_at(lv_event_list_t * list, uint32_t index);
  *   GLOBAL FUNCTIONS
  **********************/
 
+#if LV_USE_EXT_DATA
+void lv_event_desc_set_external_data(lv_event_dsc_t * dsc, void * data, void (* destructor)(void * data))
```

Comment:
On other places too, for consistency:
```suggestion
void lv_event_desc_set_external_data(lv_event_dsc_t * dsc, void * data, void (* free_cb)(void * data))
```

Reply:
Done